src/app/utils/query-url-sanitization.ts

```diff
@@ -1,67 +1,74 @@
+import { GRAPH_URL } from '../services/graph-constants';
 import { parseSampleUrl } from './sample-url-generation';
 
-// Matches patterns like "('<1f7ff346-c174-45e5-af38-294e51d9969a>')" or "('<key>')"
-const keyIdRegex = /\(\'\<?\{?[ ?0-9a-zA-Z-]*\}?\>?\'\)/g;
-
-// Matches patterns like "(query=<key>)" or "(itemat=<key>,mode=<mode>)"
-const functionParamInitialRegex = /(?<=\=).*?(?=,)/g;
-const functionParamFinalRegex = /(?<=\=).[^,]*(?=\))/g;
-
 // Matches pattterns within quotes e.g "displayName: Gupta"
 const quotedTextRegex = /"([^"]*)"/g;
 
-// Matches PII patterns
-const numberRegex =  /(?<=\/)\d+\b/g; // number between forward slashes
-const emailRegex = /([a-z0-9_\-.+]+)@\w+(\.\w+)*/gi;
-const guidRegex =
- /(\{){0,1}[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}(\}){0,1}/gi;
+// matches strings that are all alphabets
+const allAlphaRegex = /^[A-Za-z]+$/;
```

Comment:
If you use `alphabets` as the variable name, you won't need the comment above.

Reply:
I think the comment is a good clarification because there is a detail that this regex matches only a full string as opposed to succeeding on a substring match too.

Reply:
@zengin I missed the detail  in the comment and understood what the regex does after trying it out. @thewahome what do you think of adding what the regex matches  as part of the comment?

```
/**
 * will match: abc
 * won't match: ab2c
**/
```